src/routes/+layout.svelte

```diff
@@ -1,5 +1,6 @@
-<script>  
-import "../app.scss";
+<script lang="ts">  
+	import "../app.scss";
+	let { children } = $props();
 </script>
 
-<slot />
```

Comment:
good bye slot, I just think this was a dumb idea but anyway, I don't mind 3 lines of code change but why?